### PR TITLE
Port Hacker News Demo to Sample Apps

### PR DESCRIPTION
**Pull request by langlk**
Resolves [FRONTEND-439](https://algorithmia.atlassian.net/browse/FRONTEND-439)

### CODE REVIEW

File: JavaScript/hacker-news/public/js/main.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+// this API Key will only work on Algorithmia's website; get your own key at https://algorithmia.com/user#credentials
+var client = Algorithmia.client('sim3fGsmXfk6P0o1Kx8rzo2ZmAf1');
```
Comment (andryanmiller) [inline JavaScript/hacker-news/public/js/main.js]: :bulb: Let's add a quick comment here re/ where this key lives and its permissions.

**Comment by peckjon** [comment]
Please track demo keys in https://docs.google.com/spreadsheets/d/1v6D-1xyHe0VgmD32uJzvKI8XhAZH23GN1Ngxmz7MnXc/edit#gid=0 @langlk @andryanmiller

**Comment by langlk** [comment]
> Please track demo keys in https://docs.google.com/spreadsheets/d/1v6D-1xyHe0VgmD32uJzvKI8XhAZH23GN1Ngxmz7MnXc/edit#gid=0 @langlk @andryanmiller

Thanks @peckjon, added it to the sheet!

File: JavaScript/hacker-news/public/js/main.js (hunk 1 above)
Comment (langlk) [reply JavaScript/hacker-news/public/js/main.js]: As noted below, added this to the tracking sheet

**Pull request merged**